testproject/tests/test_second_step_authentication.py

```diff
@@ -652,26 +606,25 @@ def test_request_codes(active_user_with_email_otp):
 
     response = client.post(
         path="/auth/code/request/",
-        data={
-            "method": "email",
-        },
+        data={"method": "email"},
         format="json",
     )
     expected_msg = "Email message with MFA code has been sent."
     assert response.status_code == 200
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too